src/VisualStudio/Core/Def/Implementation/PickMembers/PickMembersDialogViewModel.cs

```diff
@@ -20,7 +20,7 @@ internal class PickMembersDialogViewModel : AbstractNotifyPropertyChanged
             IGlyphService glyphService,
             ImmutableArray<ISymbol> members)
         {
-            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).OrderBy(s => s.MemberName).ToList();
+            MemberContainers = members.Select(m => new MemberSymbolViewModel(m, glyphService)).ToList();
```

Comment:
@CyrusNajmabadi Can you add a comment stating the intent then? I could totally imagine somebody coming by, saying "heh, that's stupid it's not sorted", and adding in the exact thing you're removing.

(bonus points for having some sort of visible grouping so it's clear to users too.)